add items component and include it to the front end as per #48

colorlessenergy opened the pull request.

colorlessenergy committed:
add items component and include it to the frontend as per #48

planeswalker1 reviewed (changes requested):
In the container folder change the the folder name of 'items' to 'Items' to keep consistent with the folder naming convention. 

In the 'Items' file change the class name of 'Item' to 'Items' to keep consistent with the class naming convention used in the other files.

In the 'app' file change the route path of '/item' to '/items' to keep consistent with the file naming convention

colorlessenergy committed:
fixes to the review changes requested as per #49

planeswalker1 committed:
this file is not needed

This is a duplication of the other file in the folder 'Items' and it is not needed.

planeswalker1 committed:
change 'to' path in RoomsList.js

planeswalker1 reviewed (approved):
Looks good to merge and this should resolve issue #48
